### PR TITLE
Corrected grammar in documentation of GSOD

### PR DESCRIPTION
**Pull request by tripur3121**

### CODE REVIEW

File: mentorship/gsod/2023/README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
+OpenPrivacyTech is a 501(c)(3) nonprofit open-source organization dedicated to democratizing privacy technology and privacy-enhancing technologies (PETs). Our mission is to make privacy-enhancing technologies (PETs) accessible to everyone through education, research, and advocacy. Our goal is to advance the field of privacy technology by conducting research, identifying emerging trends and technologies, and developing standards and infrastructure for the adoption of privacy-enhancing technologies (PETs). We work with researchers, data scientists, developers, policymakers, industry leaders, and organizations to advocate for strong data privacy and protection laws while promoting the adoption of privacy-enhancing technologies (PETs).
```
Comment (souravcipher) [inline mentorship/gsod/2023/README.md]: I think we should keep the term Privacy Tech

Comment (tripur3121) [reply mentorship/gsod/2023/README.md]: Fair. Let me make the required changes. Thanks for the feedback.

Comment (tripur3121) [reply mentorship/gsod/2023/README.md]: @souravcipher Verify and let me know if I need to change anything else.

**Comment by tripur3121** [comment]
@souravcipher made the changes that you mentioned.

**Comment by souravcipher** [comment]
Thank you!

**Pull request merged**